admin/acceptance_tests/ogc_servers_test.py

```diff
@@ -0,0 +1,107 @@
+
+    _prefix = '/ogc_servers'
+
+    def test_index_rendering(self, test_app):
+        resp = self.get(test_app)
+
+        self.check_left_menu(resp, 'OGC Servers')
+
+        expected = [('_id_', '', 'false'),
+                    ('name', 'Name', 'true'),
+                    ('description', 'Description', 'true'),
+                    ('url', 'Basic Url', 'true'),
+                    ('url_wfs', 'WFS Url', 'true'),
+                    ('type', 'Server type', 'true'),
+                    ('image_type', 'Image type', 'true'),
+                    ('auth', 'Authentication type', 'true'),
+                    ('wfs_support', 'WFS support', 'true'),
+                    ('is_single_tile', 'Is single tile', 'true')]
+        self.check_grid_headers(resp, expected)
+
+    def test_grid_search(self, test_app):
+        # search on ogc_server name
+        self.check_search(test_app, 'server_0', 1)
+
+    def test_submit_new(self, dbsession, test_app):
+        from c2cgeoportal_commons.models.main import OGCServer
+        resp = test_app.post(
+            '/ogc_servers/new',
+            {
+                'name': 'new_name',
+                'description': 'new description',
+                'url': 'www.randomurl.com',
+                'type': 'mapserver',
+                'auth': 'No auth'
+            },
+            status=302)
+        ogc_server = dbsession.query(OGCServer). \
+            filter(OGCServer.name == 'new_name'). \
+            one()
+        assert str(ogc_server.id) == re.match('http://localhost/ogc_servers/(.*)', resp.location).group(1)
+        assert ogc_server.name == 'new_name'
+
+    def test_edit(self, test_app, ogc_server_test_data, dbsession):
+        from c2cgeoportal_commons.models.main import OGCServer
+        ogc_server = ogc_server_test_data['ogc_servers'][0]
+        resp = test_app.get('/ogc_servers/{}'.format(ogc_server.id), status=200)
+        form = resp.form
+        assert str(ogc_server.id) == self.get_first_field_named(form, 'id').value
+        assert 'hidden' == self.get_first_field_named(form, 'id').attrs['type']
+        assert ogc_server.name == self.get_first_field_named(form, 'name').value
+
+        descriptions = '{}, {}'.format(ogc_server_test_data['ogc_servers'][0].description,
+                                       ogc_server_test_data['ogc_servers'][1].description)
+        form['description'] = descriptions
+        assert form.submit().status_int == 302
+
+        assert len(dbsession.query(OGCServer).filter(OGCServer.description == descriptions).all()) == 1
+
+    def test_delete(self, test_app, ogc_server_test_data, dbsession):
+        from c2cgeoportal_commons.models.main import OGCServer
+
+        ogc_server = ogc_server_test_data['ogc_servers'][0]
+        deleted_id = ogc_server.id
+        test_app.delete('/ogc_servers/{}'.format(deleted_id), status=200)
+        assert len(dbsession.query(OGCServer).filter(OGCServer.id == deleted_id).all()) == 0
```

Comment:
or dbsession.query(OGCServer).get(deleted_id) is None
or dbsession.query(OGCServer).filter(OGCServer.id == deleted_id).count() = 0